### PR TITLE
update_yaml's

### PR DESCRIPTION
**Pull request by hyunchul386**
## Description
Update two yaml files to resolve the saber issue in the previous yaml files.
~/godas/soca/soca_3dvar_3dbump.yaml
~/godas/soca/soca_3dvar.yaml

### CODE REVIEW

File: soca/soca_3dvar.yaml [hunk 1]
```diff
@@ -117,15 +108,15 @@ cost function:
...
-        chl_min: 0.003
-        chl_max: 10.0
-        biop_min: 0.0
-        biop_max: 1.0e-6
         cicen_min: 0.05
         cicen_max: 0.05
         hicen_min: 0.1
         hicen_max: 0.1
 
+      - linear variable change name: HorizFiltSOCA
```
Comment (guillaumevernieres) [inline soca/soca_3dvar.yaml:116]: what's the reason for adding this back @hyunchul386 ?

Comment (ShastriPaturi) [reply soca/soca_3dvar.yaml:116]: @hyunchul386 can you address this?

File: soca/soca_3dvar_3dbump.yaml [hunk 2]
```diff
@@ -73,12 +74,9 @@ cost function:
...
-    - saber block name: BUMP_NICAS
-      saber central block: true
-      iterative inverse: true
-      input variables: *soca_vars
-      output variables: *soca_vars
+    saber central block:
+      saber block name: BUMP_NICAS
+      active variables: *soca_vars
       bump:
         verbosity: main
```
Comment (ShastriPaturi) [inline soca/soca_3dvar_3dbump.yaml:81]: @hyunchul386 did you run a 3dvar with this yaml?

**Comment by hyunchul386** [comment]
This yaml is already out of date.

**Pull request closed without merging**